d2bs/kolbot/libs/AutoMule.js

```diff
@@ -451,18 +451,15 @@ MainLoop:
 	},
 
 	// get a list of items to mule
-	getMuleItems: function (forceCheck = false) {
-		var item, items, forced,
-			info = this.getInfo();
+	getMuleItems: function () {
+		var item, items, info;
+
+		info = this.getInfo();
```

Comment:
is there a reason the initalization is not inline like before?

Reply:
This was the original way done by kolton. I just reverted changes but if you prefer the other way it can be done.